copy/multiple.go

```diff
@@ -29,8 +29,9 @@ const (
 )
 
 type instanceCopy struct {
-	op           instanceCopyKind
-	sourceDigest digest.Digest
+	op                            instanceCopyKind
+	sourceDigest                  digest.Digest
+	requireCompressionFormatMatch bool
```

Comment:
Non-blocking nit: Why not also call it 'forceCompressionFormat`?  I find it confusing when multiple names/terms refer to the same thing.

Reply:
SGTM done.